sdk/src/bigint.rs

```diff
@@ -1,17 +1,43 @@
 use core::{
     cmp::Ordering,
+    fmt::Debug,
     ops::{Add, BitAnd, BitOr, BitXor, Div, Mul, Neg, Not, Rem, Shl, Shr, Sub},
 };
 
 use super::{
-    env::internal::Env as _, xdr::ScObjectType, ConversionError, Env, EnvBase, EnvObj, EnvVal,
-    RawVal, TryFromVal,
+    env::internal::Env as _, xdr::ScObjectType, Binary, ConversionError, Env, EnvBase, EnvObj,
+    EnvVal, RawVal, RawValConvertible, TryFromVal,
 };
 
 #[repr(transparent)]
 #[derive(Clone)]
 pub struct BigInt(EnvObj);
 
+impl Debug for BigInt {
```

Comment:
We could add a `Display` version of this that just outputs the number without the `BigInt(` prefix and `)` suffix. cc @paulbellamy mentioned today he'd like this.

Reply:
Added.